k8s/helm/elasticsearch/values.yaml

```diff
@@ -0,0 +1,26 @@
+image: "wikibase/elasticsearch"
+imageTag: "6.5.4-wmde.1"
+imagePullPolicy: Always
+
+service:
+  type: NodePort
+  nodePort: 30000
+
+replicas: 3
+minimumMasterNodes: 2
+
+esJavaOpts: "-Xms512m -Xmx512m"
+
+resources:
+  requests:
+    cpu: "1000m"
+    memory: "2Gi"
```

Comment:
at the surface (without trying this out) a 2GB request seems quite large for something with `"-Xms512m -Xmx512m"`
The request is per pod.

Reply:
yeah good point. I'm wondering if bumping it on the other end might be better since all search might go through it at some future point in time? Or start off small and scale it up.

Reply:
The pattern I use so far is start roughly at the predicted normal usage, and then tweak as time goes by.

For the last 12 hours (probably due to no actual usage) we can see memory is within the limits
![image](https://user-images.githubusercontent.com/3308769/125954218-7e10da08-bf4c-4a95-a415-baf75725a1b9.png)